source/get-started/download-and-install.txt

```diff
@@ -4,6 +4,13 @@
 Download and Install
 ====================
 
+.. note:: Alternative Installation Methods
+
+   The following guide shows you how to install {+driver-short+} using
```

Comment:
S: the style guide is picky about "[using](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-using)," I'd change to "by using"
```suggestion
   The following guide shows you how to install {+driver-short+} by using
```